convert2rhel/actions/system_checks/convert2rhel_latest.py

```diff
@@ -243,6 +220,39 @@ def run(self):
 
         logger.info("Latest available convert2rhel version is installed.")
 
+    def _get_convert2rhel_repofile_path(self):
+        """Download the official downstream convert2rhel repofile to a temporary directory.
+
+        :return: Path of the downloaded downstream convert2rhel repofile
+        :rtype: str
+        """
+        if system_info.version.major not in C2R_REPOFILE_URLS:
+            self.add_message(
+                level="WARNING",
+                id="CONVERT2RHEL_LATEST_CHECK_UNEXPECTED_SYS_VERSION",
+                title="Did not perform convert2rhel latest version check",
+                description="Checking whether the installed convert2rhel package is of the latest available version was"
+                " skipped due to an unexpected system version",
+                diagnosis="Expected system versions: %s. Detected major version: %s"
+                % (", ".join(str(x) for x in C2R_REPOFILE_URLS), system_info.version.major),
+            )
+            return None
```

Comment:
I think, we halt the execution way before reaching any of this in case of the user having a system version that does not correspond to this, right? 

I don't mind having this defensive code, so I would leave it here